src/server/service/mail/userInvite.ts

```diff
@@ -1,6 +1,7 @@
 import { createI18nPromise } from '@i18n/i18nFactory'
 
 import { CountryIso } from '@meta/area'
+import { BasePaths } from '@meta/basePaths'
```

Comment:
What about @meta/app ?
So we'll have only one BasePaths 
Import {BasePaths} from meta/app .
What do you @sorja @rfontanarosa

Reply:
ok for me

Reply:
ok